gcp/src/ARCFileReader.cxx

```diff
@@ -124,24 +130,31 @@ ARCFileReader::ARCFileReader(const std::string &path, double jiffie_base) :
 
 
 
-ARCFileReader::ARCFileReader(const std::vector<std::string> &filename, double jiffie_base) :
-    ms_jiffie_base_(jiffie_base)
+ARCFileReader::ARCFileReader(const std::vector<std::string> &filename,
+                             unsigned experiment) : experiment(experiment)
```

Comment:
Please maintain standard white space (4-space line continuations) here and above.